### PR TITLE
Bubble_ sort

### PR DESCRIPTION
**Pull request by vicmaburrito**
-My partner and I develop different projects and put them together to create oneself
-We use methods 
-We work in pairs 
-We talked about how the code could be improved and be more professional 
-We use yield

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        array[i], array[i + 1] = array[i + 1], array[i]
+        swapped = true
+      end
+    end
+    break unless swapped
+  end
+  array
+end
+a = [1, 9, 1, 6, 7, 9, 6, 8]
+p bubble_sort(a)
```
Comment (oloomoses) [inline bubble_sort.rb]: - Kindly remove the debugging instruction as they are not required. you can read more on the effects of debugging instructions in your code  [here](https://questions.microverse.org/t/debugging-instructions-code/346)

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    end
+    j -= 1
+  end
+  arr
+end
+
+new_bubble_sort_by = bubble_sort_by(%w[hi hello hey]) do |left, right|
+  left.length - right.length
+end
+p new_bubble_sort_by
```
Comment (oloomoses) [inline bubble_sort.rb]: - Kindly remove the debugging instruction as they are not required. you can read more on the effects of debugging instructions in your code  [here](https://questions.microverse.org/t/debugging-instructions-code/346)

**Review by oloomoses — CHANGES_REQUESTED** [review]
## Hello @vicmaburrito and @diamond-nicholas,
#### Your project works well and you have met most of the requirements. Kindly fix the highlighted issues for it to be approved.
#### Feel free to reach me on slack Oloo Moses  if you have any doubts or concerns.

**Review by NtwaliHeritier — APPROVED** [review]
**STATUS: APPROVED✔**
Congratulations team. Move on with the following part👌

**Pull request merged**